lib/exception_notifier/views/exception_notifier/_environment.html.erb

```diff
@@ -1,3 +1,4 @@
+<% require 'action_dispatch/http/request' %>
```

Comment:
wdyt about taking care of this in the `EmailNotifier` class? and maybe doing `defined?(ActionDispatch::Request)` here instead https://github.com/smartinez87/exception_notification/blob/1fb43f2c7498dfba250a0bbf067fb065515c706e/lib/exception_notifier/email_notifier.rb#L34 

(same for _environment.text.erb)